include/revng/PipelineC/PipelineC.h

```diff
@@ -74,6 +74,15 @@ extern "C" {
  * Unless otherwise noted, no pointer arguments can be NULL.
  */
 
+/**
+ * Allow setting a custom traceback function
+ * this will be called just before abort()
+ * in case of an assertion failure
+ * Useful if calling from a non-C language to have
+ * Additional information
+ */
+void rp_set_custom_abort_hook(AbortHook Hook);
```

Comment:
I think those including the C header won't have access to the definition of `AbortHook`.

Reply:
copied typedef over